### ISSUE TITLE
Is this project still maintained?

### ISSUE THREAD

**Issue by miversen33** [anuvyklack/hydra.nvim#101]
There appears to be a bug in Hydra when using the newest nightly of Neovim. I was going to report said bug, but I noticed there are several bug reports, enhancements, and PRs that are open right now that have not received any interaction from the maintainer already.

Examples
- #80 
- #79 
- #95 
- #83 
- #92 
- #100 

etc

Hydra is a very popular plugin and thus if it is no longer maintained, a discussion should be started about how to correct that (be it adding maintainers who have merge permissions, getting a fork started that will be maintained, etc).

Please note, this is **not** a dig at @anuvyklack , I very much understand burn out/losing desire to work on projects/etc. I am mostly just interested in hydra not dying because nobody else can provide needed maintenance on the project.

**Comment by vsedov** [comment]
A Potential idea could be to fork the project and implement all the prs

**Comment by miversen33** [comment]
> A Potential idea could be to fork the project and implement all the prs

You're right and that is the next step as it does seem that Hydra is unmaintained now. The issue is mainly that I don't know that I am the right person to do this. I love this project and have a small bit of code in it, but I am unsure that I would be the best to maintain it long term.

That said, there is a non-0 chance that I make a fork, pull in the listed PRs for myself and allow others to submit PRs and such. I don't have any sort of org setup for that stuff yet but I don't hate the idea (especially if I add some of my existing repos into said org).

The biggest hurdle is maintainers though. I don't know that I want to maintain this on my own. And it does seem that OP is no longer maintaining it.

**Comment by kohane27** [comment]
I think the community would be happy to have @miversen33 to be the main maintainer! Given you have created import.nvim and netman.nvim. I have no experience creating nor maintaining a neovim plugin but I'm open to help out when necessary:)

**Comment by vsedov** [comment]
I think given the situation, I understand that it may not be ideal to be a full maintainer, but hydra has a few issues as of now; and it might be wise to fork [ implement prs to stabilize hydra for now ] and i agree with @kohane27. Im quite certain the community would be more than happy to have you maintain it for the time or perhaps, just implement prs.  Key aspect is to stabilize it, hopefully the rest wouldnt be too bad. 

Il be more than happy to spare some time to support maintaining this project.

**Comment by miversen33** [comment]
This will be a massive work in progress but I have started [the island of neovim misfits](https://github.com/nvim-island) which (hopefully) will be home to projects that become unmaintained over time and people can still maintain them as a group. I have no idea if this will work or not, and I have no real sense of the direction to take with it at this time.

Anyway, This org (nvim-island) is the owner of a [fork](https://github.com/nvim-island/hydra.nvim) of hydra. I have begun pulling in PRs that resolve issues and will start working on getting some other bugs patched up as I get around to them.

If anyone else is interested (please be interested) in helping to maintain this, they can submit PRs to this fork or request merge permissions on the repo of their interest on the org. Please note, I have absolutely no idea what I am doing with the org stuff so this will be a learning experience for myself as well as anyone else lol.

Anyway, hopefully hydra can live a long healthy life with more than just 1 maintainer :upside_down_face:

**Comment by miversen33** [comment]
To anyone who is reading this, the nightly issue that is causing syntax to break everywhere has been resolved on the fork. This will be my last update on this repo. I will probably be posting a call for maintainers to /r/neovim later once I have a bit more clear path on how to properly maintain this beast and the bigger beast holding it.

**Comment by hinell** [comment]
@miversen33 Maintainer is Russian, just like me, so I've tried to reach him out. Let's see if he responds.

**Comment by benlubas** [comment]
Hey everyone! We've moved to maintaining a fork under the same organization that maintains `none-ls`&mdash;the community effort to keep `null-ls` alive.

**Here's the link:** https://github.com/nvimtools/hydra.nvim

This doesn't necessarily mean that we'll take the same approach as none-ls (which is adding as many maintainers as possible), but it does mean we'll have more eyeballs and (hopefully) an easier time finding new maintainers should the need arise.

**Comment by hinell** [comment]
> @miversen33 Maintainer is Russian, just like me, so I've tried to reach him out. Let's see if he responds.

Update: he didn't respond so far. He got lost because of war probably.
